doc/python_reference.rst

```diff
@@ -274,6 +274,9 @@ Visualization
    set_3d_view
    set_3d_title
    create_3d_figure
+   check_3d_figure
+   close_3d_figure
+   take_3d_screenshot
```

Comment:
why are these exposed in the public API?

Reply:
Since I needed those features while refactoring `report.py` I thought it could be useful to have it in the public API but I can also implement them differently.

Reply:
I will remove them from the public API :)